chart-updater.py

```diff
@@ -55,6 +55,8 @@ def healthz():
 )
 @click.option("--git-ssh-identity", help="Git config SSH identity file (key).")
 @click.option("--helm-repo-url", required=True, help="Helm repo URL.")
+@click.option("--helm-repo-user", help="Helm repo user to login.")
+@click.option("--helm-repo-password", help="Helm repo user to login.", default=lambda: os.environ.get("HELM_REPO_PASSWORD", ""))
```

Comment:
NIT `help="Helm repo user to login."` The description should likely mention password?